src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3_img2img.py

```diff
@@ -684,6 +694,7 @@ def __call__(
         clip_skip: Optional[int] = None,
         callback_on_step_end: Optional[Callable[[int, int, Dict], None]] = None,
         callback_on_step_end_tensor_inputs: List[str] = ["latents"],
+        max_sequence_length: Optional[int] = None,
```

Comment:
let's use same default value here
```suggestion
        max_sequence_length: Optional[int] = 256,
```

Reply:
I didn't use the value here because if it is `None` they still get set to 256 here:
https://github.com/huggingface/diffusers/blob/5e553a7d23fce11d090ff4551a5e5c049db2f979/src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py#L208
I did it like this to change in one place in case there's the need in the future. For readability should I set it in the call too?